Code/Framework/AzFramework/AzFramework/DocumentPropertyEditor/CvarAdapter.cpp

```diff
@@ -102,12 +111,14 @@ namespace AZ::DocumentPropertyEditor
                     {
                         VectorType newContainer = Dom::Utils::ValueToType<VectorType>(value).value_or(VectorType());
                         // ConsoleCommandContainer holds string_views, so ensure we allocate our parameters here
-                        AZStd::fixed_vector<CVarFixedString, ElementCount> newValue;
+                        AZStd::fixed_vector<CVarFixedString, ElementCount + 1> newValue;
+                        newValue.push_back(functor->GetName());
```

Comment:
Just curious about pushing the functor name in as the first element. Was there a bug here before?

Reply:
It is not needed, as the AZ::IConsole has an [overload](https://github.com/o3de/o3de/blob/development/Code/Framework/AzCore/AzCore/Console/IConsole.h#L72-L88) that accepts the command name separately from the arguments.

Reply:
yep, didn't see that overload. Fixed now in several places